### PR TITLE
Update ar.yml

### PR DESCRIPTION
**Pull request by asantarissy**
An updated version of the Arabic translation.

### CODE REVIEW

File: _data/locales/ar.yml [hunk 1]
```diff
@@ -1,29 +1,30 @@
 ar:
   locale_name: العربية
-  subtitle: مدير الحزم الذي يفتقده macOS
+  subtitle: مدير الحزم الذي يفتقده macOS (أو Linux)
   pagecontent:
-    question: مالذي سيفعله Homebrew ؟
```
Comment (MikeMcQuaid) [inline _data/locales/ar.yml]: Why remove the existing translations?

Comment (asantarissy) [reply _data/locales/ar.yml]: Hey @MikeMcQuaid, I took a copy of the latest en.yml and I'm going to translate it again and refer to the old translation.

Do you have a different approach in mind?

Comment (MikeMcQuaid) [reply _data/locales/ar.yml]: Ok, cool. Was just confused, thanks.

**Comment by asantarissy** [comment]
@MikeMcQuaid I'm done here, please let me know if I missed anything.

**Comment by MikeMcQuaid** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @asantarissy!

**Comment by asantarissy** [comment]
> Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @asantarissy!

Thank you very much @MikeMcQuaid, you guys ROCK! 💪🏼

**Pull request merged**